frontend/public/components/storage-class-form.tsx

```diff
@@ -508,23 +560,54 @@ export class StorageClassForm_ extends React.Component<
     Delete: 'Delete',
   };
 
+  // Accepts a list of CSI provisioners and it checks if the
+  // provisioner is listed in CSIStorageTypes object
+  // if yes then return the provisioner with parameters that
+  // needs to be filled by user.
+  csiProvisionerMap = (csiData) => {
```

Comment:
Can't this be simplified to following ?

```
csiProvisionerMap = (csiData) => {
  csiData.map((csi) => {
    const provisionerData = _.cloneDeep(this.CSIStorageTypes[getName(csi)] || this.defaultProvisionerObj);
    provisionerData.title = getName(csi);
    provisionerData.provisioner = getName(csi);
    this.storageTypes[getName(csi)] = provisionerData;
});
```

Reply:
Nope, it will not work. As the provisioner name is prefixed with namespace name but for listedCSI I am using a non prefixed name like this `rbd.csi.ceph.com`.